network_modeling/configs/templates/1.0/aruba/full/sw-spine.secondary.j2

```diff
@@ -70,20 +72,23 @@ interface vlan {{ variables.NATIVE_VLAN }}
     active-gateway ip {{ variables.MTL_IP_GATEWAY }}
     ip helper-address 10.92.100.222
 interface vlan {{ variables.NMN_VLAN }}
+    description NMN
```

Comment:
wonder if we should think about bringing this in dynamically?  if you think so file a new ticket.